lib/galaxy_test/selenium/test_library_to_collections.py

```diff
@@ -90,7 +88,8 @@ def list_of_pairs_export(self, is_new_history=False):
         self.collection_builder_create()
         self.home()
         self.history_panel_wait_for_hid_ok(3)
-        self.history_panel_wait_for_hid_hidden(1)
-        self.history_panel_wait_for_hid_hidden(2)
+        # TODO: Check hidden datasets in new history
```

Comment:
Rebase without this? I think my PR that fixed this has been included in dev now.

Reply:
Excellent, thank you so much.